site/content/includes/installation/helm/pulling-the-chart.md

```diff
@@ -0,0 +1,12 @@
+---
+docs:
+---
+
+Pull the latest stable release of the NGINX Gateway Fabric chart:
+
+   ```shell
+   helm pull oci://ghcr.io/nginxinc/charts/nginx-gateway-fabric --untar
+   cd nginx-gateway-fabric
+   ```
+
+   If you want the latest version from the **main** branch, add `--version 0.0.0-edge` to your pull command.
```

Comment:
Why do we have all of these `includes` files? Versus just including this info in the do

Reply:
The content of the include is used in more than one document. This allows to reuse (and update) steps in one go without having to individually update each instance.